content/en/docs/tasks/administer-cluster/switch-to-evented-pleg.md

```diff
@@ -5,94 +5,104 @@ content_type: task
 weight: 90
 ---
 
-{{< feature-state for_k8s_version="v1.26" state="beta" >}}
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
 
 <!-- overview -->
 
-This page shows how to migrate notes to use event based updates for container status. The event-based
+
+This page shows how to migrate nodes to use event based updates for container status. The event-based
 implementation reduces node resource consumption by the kubelet, compared to the legacy approach
 that relies on polling.
 You may know this feature as _evented Pod lifecycle event generator (PLEG)_. That's the name used
 internally within the Kubernetes project for a key implementation detail.
 
+The polling based approach is referred to as _generic PLEG_.
+
 ## {{% heading "prerequisites" %}}
 
 * You need to run a version of Kubernetes that provides this feature.
-  Kubernetes {{< skew currentVersion >}} includes beta support for event-based container
-  status updates. The feature is beta and is disabled by default.
-  {{< version-check >}}
+  Kubernetes v1.27 includes beta support for event-based container
```

Comment:
nit: Can you use a shortcode?

Reply:
Line 8 has the shortcode you are looking for. The explicit callout of this version number is because the behavior is against the tradition. The feature is Beta, but it is disabled by default.